minio/rockcraft.yaml

```
@@ -0,0 +1,80 @@
+# Based on: https://github.com/minio/minio/blob/RELEASE.2021-09-03T03-56-13Z/Dockerfile.release
+name: minio
+summary: MinIO is a High Performance Object Storage, API compatible with Amazon S3 cloud storage service.
+description: |
+  MinIO object storage is fundamentally different. Designed for performance and the S3 API, it is 100% open-source. MinIO is ideal for large, private cloud environments with stringent security requirements and delivers mission-critical availability across a diverse range of workloads.
+version: 21.09.03
```

For this image we are following the format `ckf-<ckf release number>`

```suggestion
version: ckf-1.9
```

---

Fixed